common/etc/nginx/include/utils.js

```diff
@@ -0,0 +1,62 @@
+/*
+ *  Copyright 2023 F5, Inc.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+/**
+ * Flag indicating debug mode operation. If true, additional information
+ * about signature generation will be logged.
+ * @type {boolean}
+ */
+const DEBUG = parseBoolean(process.env['S3_DEBUG']);
+
+
```

Comment:
Why did the `parseArray` function not get moved to the `utils.js` file?

Reply:
- Thanks @dekobon for your question. 
- Because it is not related to this PR to read and write credentials. So, I was planning to move others to the `utils.js` later to minimize changes per each PR with avoiding any mistakes as much as I can. 
- But, it makes sense to move to the `utils.js`. Hence, I have moved the function in the common utils. 😸